src/system/components/sente.cljc

```diff
@@ -67,7 +67,12 @@
          (sente/chsk-disconnect! chsk))
        (when-let [stop-f router]
          (stop-f))
-       (dissoc component :router :chsk :ch-recv :chsk-send! :chsk-state))))
+       (assoc component
+              :router nil
+              :chsk nil
+              :ch-recv nil
```

Comment:
@danielsz reading through the code I don't see where this `:ch-recv` key is coming from, on the other hand the `:ch-chsk` key is not being reset here; maybe a spelling mistake?